java/avro-converter/src/test/java/com/microsoft/azure/schemaregistry/kafka/connect/avro/AbstractKafkaSerdeConfigTest.java

```diff
@@ -0,0 +1,30 @@
+package com.microsoft.azure.schemaregistry.kafka.connect.avro;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.util.Collections;
+import org.junit.jupiter.api.Test;
+import com.azure.core.credential.TokenCredential;
+
+public class AbstractKafkaSerdeConfigTest {
```

Comment:
would it be possible to add more testcases? I can understand it might be complex to do E2E testing but at least just exercising the converter API with some of the conversion scenario would be good - some of my comments would have be covered by the unit testing I think.